R/recorder.R

```diff
@@ -131,3 +131,30 @@ recordTest <- function(app = ".", save_dir = NULL, load_mode = FALSE, seed = NUL
   invisible(res$file)
 }
 
+
+#' Register an input processor for the test recorder
+#'
+#' @description
+#' `registerInputProcessor()` registers an input processor which will be used by
+#' the test recorder. The input processor function should take one parameter,
+#' `value`, and return a string of R code which returns the desired value.
```

Comment:
Looks like the function takes multiple arguments? https://github.com/rstudio/learnr/pull/407/files#diff-e203eb3513bb0129d101f49b354716bdR7

Reply:
Also, does it have to be a string? Or can it be a character vector?

Reply:
The `value` is an R object which is the value of, say, `input$x`. It could potentially be any R object (in the case of learnr code, it's a list).